.github/workflows/ci.yml

```
@@ -168,3 +168,19 @@ jobs:
         run: devcontainer build --image-name devcontainer-fdo-rs .
       - name: Test building in devcontainer
         run: docker run --rm -v `pwd`:/code:z --workdir /code --user root devcontainer-fdo-rs cargo build --verbose
+
+  local_packaging:
+    name: Test local packaging
+    runs-on: ubuntu-latest
+    container: fedora:latest
+    steps:
+      - name: Install deps
+        run: |
+          dnf install -y rust-packaging python3-docutils fedora-packager bzip2 cargo git make gcc openssl openssl-devel findutils golang git tpm2-tss-devel swtpm swtpm-tools git clevis clevis-luks cryptsetup cryptsetup-devel clang-devel cracklib-dicts
+      - name: Package
+        run: |
+          git clone https://github.com/fedora-iot/fido-device-onboard-rs.git
+          pushd fido-device-onboard-rs
+          make rpm
+          if [[ $(ls rpmbuild/RPMS/x86_64/*.rpm | wc -l) -eq 16 ]]; then rm -rf rpmbuild; else exit 1; fi
```

```suggestion
          if [[ $(ls rpmbuild/RPMS/x86_64/*.rpm | wc -l) -lt 16 ]]; then exit 1; fi
          dnf install -y rpmbuild/RPMS/x86_64/*.rpm
```

This would make sure to also actually test the RPMs for installability.

---

Thanks for that, will change it in a minute :+1: